rpmlint/checks/SpecCheck.py

```diff
@@ -21,8 +13,8 @@
 
 
 def re_tag_compile(tag):
-    r = r'^%s\s*:\s*(\S.*?)\s*$' % tag
-    return re.compile(r, re.IGNORECASE)
+    rpm_tag = r'^%s\s*:\s*(\S.*?)\s*$' % tag
```

Comment:
use format instead of % syntax if you are already rewritting the line.

Reply:
Thanks for the suggestion, I made the necessary changes in the most recent commit. Check it out [here](https://github.com/thisisshub/rpmlint/commit/153d7211e6afabe5bb938012cf99a60cc2897e21)